sockeye/inference.py

```diff
@@ -1250,9 +1238,8 @@ def _make_result(self,
         :param translation: The translation + attention and score.
         :return: TranslatorOutput.
         """
-        # remove special sentence start symbol (<s>) from the output:
```

Comment:
the docstring of this class method needs to be updated

Reply:
I think it's already correct?

Reply:
I mean line 1236

Reply:
Still don't understand. The comment now reads:

```
        Returns a translator result from generated target-side word ids, attention matrix, and score.
        Strips stop ids from translation string.

        :param trans_input: Translator input.
        :param translation: The translation + attention and score.
        :return: TranslatorOutput.
```

which is correct.

Reply:
True, your are correct. I thought the stop id stripping was removed.

Reply:
Okay, cool. I had been willing to be 2:1 that you were right :)

Reply:
I should have insisted on it just a little bit more :D